cmd/tidb-server/BUILD.bazel

```diff
@@ -10,6 +10,7 @@ go_library(
         "//pkg/config",
```

Comment:
Can you add some test cases for it?

Reply:
I am not sure exactly what you mean, but we are in the process of creating tests for this code.